import_export/resources.py

```diff
@@ -579,7 +579,7 @@ def get_diff_headers(self):
         """
         Diff representation headers.
         """
-        return self.get_user_visible_headers()
+        return [force_str(field.column_name) for field in self.get_import_fields()]
```

Comment:
Ahh I see, so it used get_user_visible_headers which used get_fields instead of get_import_fields. Nice :)